SchoolProject/src/main/java/Interfaces/IMenuCommand.java

```diff
@@ -0,0 +1,12 @@
+package Interfaces;
+
+import java.util.List;
+
+import models.Student;
+import models.Subject;
+
+
+public interface IMenuCommand {
```

Comment:
The idea to have a command for menu options is ok, but remember that command contains only an execute method, so probably the classes that will implement it, needs to overwrite the execute method